scripts/talos/commitChanges/index.js

```diff
@@ -0,0 +1,8 @@
+const runExec = require('../../utilities/runExec');
+
+const commitChanges = message =>
+  runExec({ command: `git add packages` })
```

Comment:
Wouldn't `git add packages` miss the main package.json? Which should also be bumped with the package version numbers

Reply:
Yeah the main body of this PR was done before we moved all the psammead package dev dept to root. I was going to raise and issue to bump the base one after this is merged, but I can do it now if you think its high enough importance?